ext/mixed/js/text-scanner.js

```diff
@@ -60,6 +60,9 @@ class TextScanner extends EventDispatcher {
             return;
         }
 
+        const modifiers = DOM.getActiveModifiers(e);
+        this.trigger('updateActiveModifiers', {modifiers});
```

Comment:
Rename event to be more event-like: `activeModifiersChange` or `activeModifiersChanged`

(I forget if we have a consistent pattern for tense on event names; I have the feeling we don't.)

Reply:
I've seen both, but I renamed it to the past tense https://github.com/FooSoft/yomichan/pull/487/commits/28afafd568a8e58d86c96ea38edc1ff8093ec94e